Pre loaded config import fails with webpack dynamic module parsing

### Issue Summary
Using the [`PRE_LOADED_CONFIG`](https://boxyhq.com/docs/jackson/pre-loaded-configuration) fails to load the config js file.  
```
config api error: Error: Cannot find module '/Users/xxxxx/Code/jackson/_config/boxyhq.js'
``` 

### Steps to Reproduce

1. Run jackson with PRE_LOADED_CONFIG set to point to a local dir
2. Invoke the GET config 
3. Requests fails with above error


### Technical details

Summarized nicely at https://stackoverflow.com/a/42804941/3939061

> You cannot use a variable as argument to require. Webpack needs to know what files to bundle at compile time. As it does no program flow analysis, it can't know what you pass to the function. In that case it might be obvious, but this could go as far as using user input to decide what module to require, and there is no way webpack can possibly know which modules to include at compile time, so webpack does not allow it.

--- a/npm/src/read-config.ts
+++ b/npm/src/read-config.ts
@@ -13,8 +13,9 @@ const readConfig = async (preLoadedConfig: string): Promise<IdPConfig[]> => {
   for (const idx in files) {
     const file = files[idx];
     if (file.endsWith('.js')) {
-      // eslint-disable-next-line @typescript-eslint/no-var-requires
-      const config = require(path.join(preLoadedConfig, file)) as IdPConfig;
+      const { default: config }: { default: IdPConfig } = await import(
+        /* webpackIgnore: true */ path.join(preLoadedConfig, file)
+      );
       const rawMetadata = await fs.promises.readFile(
         path.join(preLoadedConfig, path.parse(file).name + '.xml'),
         'utf8'

